app/controllers/api/v2/congress_controller.rb

```diff
@@ -1,21 +1,33 @@
 class Api::V2::CongressController < ApplicationController
 
   def zipcode_with_quiz
-    representatives = Representative.from_zip(params[:zipcode])
-    quiz = parse_quiz(params)
-    render json: {
-                    politicians: RepresentativeSerializer.new(representatives, {params: quiz}),
-                    summary_statistics: SummaryStatisticsSerializer.new(VoteSmartFacade.summary_statistics(representatives, quiz))
-                  }
+    if params[:aclu].nil?
+      render json: {error: 'please include quiz!'}, status: :bad_request 
+    elsif params[:zipcode].blank? 
+      render json: {error: 'please include zipcode!'}, status: :bad_request 
+    else 
+      representatives = Representative.from_zip(params[:zipcode])
+      quiz = parse_quiz(params)
+      render json: {
+                      politicians: RepresentativeSerializer.new(representatives, {params: quiz}),
```

Comment:
multiple serializers are an effective way to break up the functionality of the JSON response.